client/app/ui-components/dual-list/dual-list.component.js

```diff
@@ -0,0 +1,21 @@
+angular.module("managerApp").component("cuiDualList", {
+    templateUrl: "app/ui-components/dual-list/dual-list.html",
+    controller: "DualListCtrl",
+    controllerAs: "$ctrl",
+    bindings: {
+        sourceListLabel: "@",
```

Comment:
This is quite a big list of bindings :/

Reply:
Some of them, related to translations, should be only in the provider
moveAllLabel,
removeAllLabel,
sourceListEmptyLabel,
..

Reply:
I agree with tremblaymath